Manifests/ManifestsApple/com.apple.applicationaccess-macOS.plist

```diff
@@ -938,6 +938,18 @@ Additionally, if set to true and ScreenObservationPermissionModificationAllowed
 				<key>pfm_type</key>
 				<string>boolean</string>
 			</dict>
+			<dict>
+				<key>pfm_default</key>
+				<true/>
+				<key>pfm_macos_min</key>
+				<string>13.0</string>
+				<key>pfm_name</key>
+				<string>allowBluetoothModification</string>
```

Comment:
Since we added a key, let's bump the `pfm_version` of this manifest

Reply:
Addressed.